website/docs/user-group-role/user/user_basic_operations.md

```diff
@@ -4,6 +4,8 @@ title: Manage users
 
 The following topics are for the basic management of users: how to create, modify, delete or deactivate users, and using a recovery email.
 
+[Policies](../../policies/index.md) can be used to further manage how users are authenticated. For example, in authentik email addresses are (by default) not required to be unique, but you can use a policy to [enforce unique email addresses](../../policies/working_with_policies/unique_email.md).
```

Comment:
"in authentik email addresses" looked like one whole phrase about "authentik email addresses."  Suggestion: "For example, by default authentik does not require email addresses be unique, but you can use a policy to ..."  

Our lack of capitalization of our name is a partial weakness, as it doesn't set us off as a proper noun the way we're trained to read. It sometimes takes a second read to tease apart the implied meaning.